File: MODUL-6/c/PRAK603-2310817210007-RAYMONDHARIYONO.c
#include <stdio.h>

int main(){

    int n1, n2;

    scanf("%d%d",&n1,&n2);

    if(n1 == n2){
        int arr1[n1];
        int arr2[n2];

        for(int i=0; i < n1; i++){
            scanf("%d",& arr1[i]);
        }

        for(int i=0; i < n2; i++){
            scanf("%d",& arr2[i]);
        }

        for(int i=0; i < n1 && i < n2; i++){
            int hasil = arr1[i] * arr2[i];
            printf("%d ", hasil);
        }
        printf("\n");
    }  
     else{
        printf("Jumlah tidak sama");
     }
return 0;
}

File: MODUL-6/python/PRAK603-2310817210007-RAYMONDHARIYONO.py
jumlahBaris = 2
banyakAngka, banyakAngkaBaris = map(int, input().split())
if banyakAngka == banyakAngkaBaris:
    arr1 = []

    for i in range(jumlahBaris):
        arr1.append(list(map(int, input().split())))
        
    for i in range(jumlahBaris - 1):
        for j in range(banyakAngka):
            if i + 1 < jumlahBaris:
                arr1[i][j] = arr1[i][j] * arr1[i + 1][j]

    for i in range(jumlahBaris - 1):
        for j in range(banyakAngka):
            print(arr1[i][j], end=" ")

else:
    print("Jumlah Tidak Sama")